data/json/items/melee/bludgeons.json

```diff
@@ -1764,5 +1764,64 @@
     "price": 16000,
     "qualities": [ [ "HAMMER", 1 ] ],
     "melee_damage": { "bash": 22, "stab": 23 }
+  },
+  {
+    "type": "GENERIC",
+    "id": "exodii_club",
+    "name": "crusher",
+    "description": "Resembling a tetsubo or baseball bat studded with metal spikes, this is a simple and brutal weapon used by Exodii when the ammo runs low. The massive weight makes this a daunting weapon to wield for those without artificial muscles.",
+    "weight": "10 kg",
+    "min_strength": 12,
+    "price_postapoc": 400,
+    "color": "grey",
+    "symbol": "|",
+    "material": "qt_steel",
+    "techniques": [ "WBLOCK_1", "SPIN", "WIDE", "BRUTAL", "SWEEP" ],
+    "flags": [ "DURABLE_MELEE", "SHEATH_SPEAR", "ALWAYS_TWOHAND" ],
+    "weapon_category": [ "MACES", "GREAT_HAMMERS" ],
+    "volume": "2250 ml",
+    "longest_side": "120 cm",
+    "to_hit": { "grip": "weapon", "length": "long", "surface": "any", "balance": "uneven" },
+    "qualities": { "HAMMER": 1 },
+    "melee_damage": { "bash": 40, "pierce": 20 }
+  },
+  {
+    "id": "exodii_powerfist",
+    "type": "ARMOR",
+    "name": { "str_sp": "hammerhand" },
+    "description": "A weapon used by bipedal Exodii when the ammo runs low. Delivering a punch with this glove triggers a pneumatic hammer that slams into the target. A built-in compressor requires a few seconds between strikes.",
+    "weight": "3 kg",
+    "volume": "300 ml",
+    "longest_side": "50 cm",
+    "price": 1100,
+    "price_postapoc": 250,
+    "to_hit": { "grip": "weapon", "length": "hand", "surface": "any", "balance": "neutral" },
+    "material": [ "qt_steel" ],
+	"sided": "true",
+    "symbol": "[",
+    "color": "light_gray",
+    "power_armor": true,
+    "flags": [ "WATER_FRIENDLY", "DURABLE_MELEE", "OVERSIZE", "OUTER" ],
+    "armor": [
+      {
+        "material": [
+          { "type": "qt_steel", "covered_by_mat": 100, "thickness": 2 },
+        ],
+        "covers": [ "hand_l", "hand_r" ],
+        "coverage": 100,
+        "encumbrance": 5
+      }
+    ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE",
+        "airtight": true,
+        "rigid": true,
+        "watertight": true,
+        "ammo_restriction": { "oxygen": 100 }
+      }
+    ],
+    "melee_damage": { "bash": 50, "cut": 10 },
```

Comment:
But it doesn't consume any charged to make such damage, yes? I don't see any use_actions or some flag, that prevent usage of the weapon without charges

Reply:
Doesn't yet, just wanted to get this draft put up so I don't lose track.

Reply:
hmmm, can we make it as technique with `"needs_ammo": true,`, and item uses this techniques? so technique is used automatically if there is a charge, and item is regenerate 1 change every X seconds?